packages/cadl-ranch-specs/http/projection/projected-name/main.tsp

```diff
@@ -128,3 +128,51 @@ op parameter(
   @query
   `default-name`: string
 ): NoContentResponse;
+
+@route("/model")
+@operationGroup
+namespace Model {
+  @projectedName("client", "_DontUseMeAsAName_") // Should be ignored
+  @projectedName("csharp", "CSModel")
+  @projectedName("java", "JavaModel")
+  @projectedName("javascript", "TSModel")
+  @projectedName("python", "PythonModel")
+  model ModelWithLanguageProjectedName {
+    @doc("Pass in true")
+    defaultName: boolean;
```

Comment:
Why do we need this? Any use in test?

Reply:
This is not used in test. But spread an empty model in operation, compiler will consider no body parameter. So the model will not be referenced in operation.